docs/pages/server-access/includes/ssh-hardening.mdx

```diff
@@ -0,0 +1,43 @@
+## OpenSSH hardening
+
+This section describes general ways to secure your OpenSSH client and server connections. 
```

Comment:
This addresses some of the #6958 items. I can add to this now, add to this once more 7.0 features are released, etc.